src/witan/models/dem/ccm/schemas.clj

```diff
@@ -26,47 +25,72 @@
 ;;For core CCM projection loop
 (def HistBirthsEstSchema
   (make-ordered-ds-schema [[:gss-code s/Str] [:district s/Str] [:sex s/Str] [:age s/Int]
-                           [:var s/Str] [:year s/Int] [:estimate s/Num]]))
+                           [:var s/Str] [:year s/Int] [:estimate double]]))
```

Comment:
they were added at different times and we must have missed that they were the same- have gotten rid of this one now and am only using ComponentMYESchema.